src/SImpl.CQRS.OpenTelemetry/Module/SqrsOpenTelemetryModule.cs

```diff
@@ -0,0 +1,51 @@
+using Microsoft.Extensions.DependencyInjection;
+using SImpl.CQRS.Commands;
+using SImpl.CQRS.Events;
+using SImpl.CQRS.OpenTelemetry.Services.Commands;
+using SImpl.CQRS.OpenTelemetry.Services.Events;
+using SImpl.CQRS.OpenTelemetry.Services.Queries;
+using SImpl.CQRS.Queries;
+using SImpl.Modules;
+
+namespace SImpl.CQRS.OpenTelemetry.Module;
+
+public class SqrsOpenTelemetryModule(SqrsOpenTelemetryModuleConfig config) : IServicesCollectionConfigureModule
```

Comment:
Is this not meant to be CqrsOpenTelemetryModule (and the same for the config class)?